Sources/SQLiteNIO/SQLiteError.swift

```diff
@@ -54,6 +55,33 @@ public struct SQLiteError: Error, CustomStringConvertible, LocalizedError {
         case warning
         case row
         case done
+        
+        // SQLite "extended" result codes
+        case errorMissingCollatingSequence, errorRetry, errorMissingSnapshot
```

Comment:
Urgh, we need a blanket ban on enums in the next major releases

Reply:
Or, you know, the core team could finally listen to people about the need to solve this problem 🤨